packages/api/test/e2e/queries.e2e.ts

```diff
@@ -136,7 +136,7 @@ describe('e2e queries', () => {
   describe('Staking account derived query', () => {
     it("Get staking account details", async done => {
       const stashId = '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY'; // alice_stash
-      const stakingAccount = await api.derive.staking.queryStakingAccountDetails(stashId);
+      const stakingAccount = await api.derive.staking.stakingAccountDetails(stashId);
```

Comment:
`api.derive.staking.stakingAccountDetails` is abit clunky with `staking.staking`

would be much nicer as something like `api.derive.staking.accountInfo`

Reply:
done